src/3d/shaders/lines.frag

```diff
@@ -0,0 +1,30 @@
+#version 150
```

Comment:
Should these files have license headers too?

Reply:
Good question... just checked Qt sources and they did not include any license headers in shaders. I'm quite neutral in this regard. Two small advantages when they are left out: 1. shader compilation is done in run-time, so less amount of text to compile, 2. on shader compilation error the shader is printed along with the error, so extra license headers can make the output unnecessarily verbose.

Reply:
Also no strong opinion here, but these are 2 good reasons not to include them.